### DISCUSSION TITLE
Newbie - Conn 5 and 8, orig. Open Deck Board, Function? -> SOLVED!

### DISCUSSION THREAD

**Discussion by Parathym** [shanteacontrols/OpenDeck#126] [Q&A] [answered]
![OPen Deck Conn](https://user-images.githubusercontent.com/97341571/148642754-a33bdb51-9579-4915-aa25-fb549ba15464.JPG)

I need help: What are the yellow marked connections for (Conn 8 and 5, original Open Deck Board, latest version)? 
Can`t find any info in the Wiki...
Thanks, Parathym

**Comment by paradajz** [comment, marked as answer]
First one is for programming the board via external programmer such as STLink.
Second one is for SPI, but it is unused.

**Reply by Parathym** [reply]
Wow - thanks for your fast reply!
